### PR TITLE
Add CIRRUS_URL param to strato-run.sh

### PR DESCRIPTION
**Pull request by samrit**

### CODE REVIEW

File: strato-run.sh [hunk 1]
```diff
@@ -78,6 +78,7 @@ then
     export BLOC_DOC_URL=${BLOC_DOC_URL:-http://localhost/docs/?url=/bloc/v2.1/swagger.json}
     export STRATO_URL=${STRATO_URL:-http://localhost/strato-api/eth/v1.2}
     export STRATO_DOC_URL=${STRATO_DOC_URL:-http://localhost/docs/?url=/strato-api/eth/v1.2/swagger.json}
+    export CIRRUS_URL=${CIRRUS_URL:-http://localhost/cirrus/search}
```
Comment (toddmohney) [inline strato-run.sh]: Kind of a shame that we have both `CIRRUS_URL` and `cirrusurl` as env vars. 
Does is make sense that they have different values? 
If that does make sense, is there a better name for them to clarify what they're for and why they're different?

**Comment by toddmohney** [comment]
@samrit can you take a look at the value of `CIRRUS_URL` on my p2p branch and push any necessary changes there as well?
https://github.com/blockapps/strato-getting-started/blob/p2p-testing-2/strato-run.multinode.sh#L82

File: strato-run.sh (hunk 1 above)
Comment (samrit) [inline strato-run.sh]: Agreed. @nikitamendelbaum and I talked about this last month but decided to let the environment variable be for now to reduce moving parts. But there definitely should only be one variable.

Comment (nikitamendelbaum) [inline strato-run.sh]: Var names come from different services and yes, they have the different values. It's kinda in todo list currently.

**Pull request merged**